pkg/cluster/px/px.go

```diff
@@ -769,22 +769,26 @@ func (ops *pxClusterOps) isScaleDownOfSharedAppsRequired(isMajorVerUpgrade bool,
 	return opts.SharedAppsScaleDown == SharedAppsScaleDownOn
 }
 
-// isMajorVersionUpgrade checks if this is a 1.2 to 1.3/1.4 upgradee
-func (ops *pxClusterOps) isMajorVersionUpgrade(spec *apiv1alpha1.Cluster) (bool, error) {
+// doesUpgradeNeedAppDrain checks if target is 1.3.3 or upgrade is from 1.2 to 1.3/1.4
+func (ops *pxClusterOps) doesUpgradeNeedAppDrain(spec *apiv1alpha1.Cluster) (bool, error) {
```

Comment:
Unconventional naming -- methods that return boolean should be called "is*()", as they are in the rest of the code.
Please rename to `isUpgradeAppDrainRequired()` or similar.

Reply:
done